docs/Using-Fleet/ChromeOS.md

```diff
@@ -7,6 +7,8 @@ Fleet provides a Chrome extension which you can install via Google Admin.
 
 To learn how to add ChromeOS hosts to Fleet, visit [here](https://fleetdm.com/docs/using-fleet/adding-hosts#add-chromebooks-with-the-fleetd-chrome-extension).
 
+> The fleetd Chrome browser extension is supported on ChromeOS operating systems that are managed using [Google Admin](https://admin.google.com). It is not intended for non-ChromeOS hosts with the Chrome browser installed.
```

Comment:
Can we consolidate this with the blockquote above it?

<img width="730" alt="Screenshot 2023-06-30 at 7 47 57 PM" src="https://github.com/fleetdm/fleet/assets/3065949/2de41bd5-aea5-4d60-a6e9-44efac0db539">

Reply:
yes, thank you!